.eslintrc

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
awesome job! 💯 
I hope that you share required packages or add those to package.json to work this `.eslintrc` 🙏